### ISSUE TITLE
Experience based flight positions

### ISSUE THREAD

**Issue by KeeseAF** [dcs-liberation/dcs_liberation#1541]
I have this idea that co-relates to the skill leveling feature within Operation LIberation.

TLDR: The more experience you have, the more flights you lead/fly

With flying with ai skill leveling on, I try to fly with the same ai wingmen in order to train them. This way they get better before they go off and fly with others. 

However with the current system, the auto planner doesn'ttake in to account who has more experience when giving out assignments.

Im proposing that the autoplanner prioritizes the ai wingmen who have more flights, and/or difficulty rating to LEAD the flights that they are assigned to. And If they have a succesful mission ( they survive and rtb successfully) they could get a buff in terms of XP boost for flying with an experienced Flight Lead.

**Comment by KeeseAF** [comment]
I forgot to add, we could possibly sprinkle various amounts of experinced pilots in all the squadrons to help train the newer pilots.

**Comment by KeeseAF** [comment]
Having a Mission Skill level requirement for certain missions.
 IE. 
Anybody who is at least SK Level of Good or higher can do CAS Missions.
Anybody SK Level high or higher can do Escort Missions.
